statsmodels/tools/tools.py

```diff
@@ -511,3 +513,36 @@ def __init__(self, **kw):
                       message='webuse will be removed from the tools '
                               'namespace in the 0.7.0 release. Please use the'
                               ' new import.')
+
+
+def _ensure_2d(x, ndarray=False):
```

Comment:
I think you need 3 settings for ndarray option

in the models we need "always", i.e. independent of the input `x` we need a ndarray for further processing